roles/engineblock/templates/engine.conf.j2

```diff
@@ -48,4 +48,14 @@ Listen {{ apache_app_listen_address.engine }}:{{ loadbalancing.engine.port }}
     Include ssl_backend.conf
     {% endif %}
 
+    <IfModule mod_expires.c>
+      ExpiresActive on
+      ExpiresByType image/x-icon "access plus 1 year"
+      ExpiresByType font/woff2 "access plus 1 year"
+      ExpiresByType text/css "access plus 1 year"
+      ExpiresByType text/js "access plus 1 year"
+      ExpiresByType image/png "access plus 6 months"
```

Comment:
Or `image/*` `font/*`? Or is that too bold?

Reply:
Bold is better! Going to test if this introduces additional issues on EB. I don't think so, otherwise we would have noticed with the inspections..